multi-int-store/src/main/java/com/connexta/multiintstore/controllers/StoreController.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.multiintstore.controllers;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequestMapping("/mis")
+public class StoreController {
+  @PutMapping(value = "/product/{resourceId}", consumes = "*/*")
```

Comment:
❓ what does the `consumes = "*/*"` string do?

Reply:
It means the endpoint is capable of accepting all mediatypes. I'm sure it will change when we implement the API.